scripts/scil_visualize_bundlesList_mosaic.py

```diff
@@ -0,0 +1,229 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+import argparse
+from distutils.version import LooseVersion
+import os
+import shutil
+
+from dipy.viz import actor, window
+import nibabel as nib
+
+if LooseVersion(nib.__version__) < LooseVersion('2.1.0'):
```

Comment:
Is this still necessary, it has been more than a year since that release.

Reply:
No, in the requirements, we grep the 2.3 version

Reply:
Ok, removed